sdk/communication/azure-communication-chat/azure/communication/chat/_shared/auth_policy_utils.py

```diff
@@ -32,17 +32,19 @@ def get_authentication_policy(
     :return: Either AsyncBearerTokenCredentialPolicy or BearerTokenCredentialPolicy or HMACCredentialsPolicy
     :rtype: ~azure.core.pipeline.policies.AsyncBearerTokenCredentialPolicy or
     ~azure.core.pipeline.policies.BearerTokenCredentialPolicy or
-    ~azure.communication.chat.shared.policy.HMACCredentialsPolicy
+    ~azure.communication.callautomation.shared.policy.HMACCredentialsPolicy
```

Comment:
This is an internal method - so we don't need a fully completed docsting. Can you remove the `:rtype:` statement here? The `:return:` statement should be sufficient.